go/tango/main.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import (
+	"gitea.com/lunny/tango"
+)
+
+type Params struct {
+	tango.Params
+}
+
+func (req *Params) Get() string {
+	return req.Params.Get(":id")
+}
+
+type Empty struct{}
+
+func (Empty) Get() string {
+	return ""
+}
+
+func (Empty) Post() string {
+	return ""
+}
+
+func main() {
+	t := tango.Classic()
```

Comment:
I suggest to use 
```
t := tango.New(
		tango.Logging(),
		tango.Recovery(false),
		tango.Return(),
		tango.Param(),
	)
```

And the default port is 8000.

Reply:
sure :tada: 

However, the port is set by an environment variable